server/src/server.ts

```diff
@@ -1,15 +1,17 @@
 import express, { Application, Request, Response } from "express";
 import cors from "cors";
+import { serverConfig } from "./config";
 
+const port = serverConfig.port;
+const hostname = serverConfig.hostname;
 const app: Application = express();
 app.use(express.json());
 app.use(cors());
-const port = 3000;
 
 app.get("/", (_req: Request, res: Response): void => {
   res.send("Hello there!");
 });
 
 app.listen(port || 5000, (): void => {
```

Comment:
Nitpick: this default is no longer necessary.

Reply:
That was some bug on Github. That shouldn't be there. I can't even see it on my part nor couldn't. 🤷🏻‍♂️